data/filter/wg21.py

```diff
@@ -228,10 +246,19 @@ def build_row(elems):
 
     header = pf.Null()
     width = 0
+    caption = pf.Null()
+    captionsFound = 0
     table.content.append(pf.HorizontalRule())
     for elem in table.content:
         if isinstance(elem, pf.Header):
             header, width = build_header(elem)
+        elif isinstance(elem, pf.Para):
+            caption = elem
+            captionsFound = captionsFound + 1
+        elif isinstance(elem, pf.Table):
```

Comment:
This is for... an embedded table within a tony table? I'm not really following why this is here. Could you elaborate on this?

Reply:
the embedded table is an ugly hack used to leverage the caption parsing in Table. The content of an embedded table is ignored. only the caption is extracted.

Reply:
Agreed offline to use `pf.BlockQuote` instead of `pf.Para` and to remove the `pf.Table` hack.

Reply:
I will remove the hack.